rails-commenting.rb

```diff
@@ -6,25 +6,25 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# 1)
+# 1) The BlogPostsController is the controller for all of the methods to be used in the application so we can use it on the web application.
 class BlogPostsController < ApplicationController
   def index
-    # 2)
+    # 2) @posts is an instance variable setup inside of our index(which routes as a get) to access all of our BlogPosts models previously made.
     @posts = BlogPost.all
   end
 
   def show
-    # 3)
+    # 3) @post is an instance variable setup inside of our show(Which routes as a get) to access only a single model in BlogPost using .find and the params that recieves info from the rails server of an :id.
```

Comment:
Yup, you go it! You are accessing one instance of a class and passing params id as an argument to find the specific one we want